src/wp-includes/class-wp-theme-json.php

```diff
@@ -1216,11 +1648,15 @@ public static function get_from_editor_settings( $settings ) {
 			$theme_settings['settings']['typography']['fontSizes'] = $font_sizes;
 		}
 
+		// This allows to make the plugin work with WordPress 5.7 beta
```

Comment:
Doesn't make sense to refer to "the plugin" in Core.

Reply:
https://github.com/WordPress/wordpress-develop/pull/1808/commits/01a8bad10bc1327d008e567fad9bbd7c391b9fd6